tests/Events/QueuedEventsTest.php

```diff
@@ -339,9 +339,9 @@ public function handle()
         //
     }
 
-    public function viaQueue($event)
+    public function viaQueue($event = null)
```

Comment:
If event is now sometimes `null`, isn't this a breaking change?

Reply:
The `$event` argument defaulting to null only impacts the code inside the `viaQueue` function of the `TestDispatcherGetQueueDynamically` class, which is also changed for proper support. Since the `viaQueue` function doesn't specify any arguments normally and this seems to be a case that is used in the tests only, I don't see this as a breaking change.

If you do, please share why and I will take a look and adjust my pull request.

Thanks for reviewing.